ocs_ci/templates/workloads/jenkins/jenkins_template

```diff
@@ -0,0 +1,246 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+labels:
+  app: jenkins-persistent-ocs
+  template: jenkins-persistent-ocs-template
+message: A Jenkins service has been created in your project.  Log into Jenkins with
+  your OpenShift account.  The tutorial at https://github.com/openshift/origin/blob/master/examples/jenkins/README.md
+  contains more information about using this template.
+metadata:
+  annotations:
+    description: |-
+      Jenkins service, with persistent storage.
+
+      NOTE: You must have persistent volumes available in your cluster to use this template.
+    iconClass: icon-jenkins
+    openshift.io/display-name: Jenkins
+    openshift.io/documentation-url: https://docs.okd.io/latest/using_images/other_images/jenkins.html
+    openshift.io/long-description: This template deploys a Jenkins server capable
+      of managing OpenShift Pipeline builds and supporting OpenShift-based oauth login.
+    openshift.io/provider-display-name: Red Hat, Inc.
+    openshift.io/support-url: https://access.redhat.com
+    samples.operator.openshift.io/version: 4.4.0-0.nightly-2020-06-08-083627
+    tags: instant-app,jenkins
+  creationTimestamp: "2020-06-10T17:53:53Z"
+  labels:
+    samples.operator.openshift.io/managed: "true"
+  name: jenkins-persistent-ocs
+  namespace: openshift
+  resourceVersion: "15222"
+  selfLink: /apis/template.openshift.io/v1/namespaces/openshift/templates/jenkins-persistent-ocs
+  uid: 3965f694-f275-441d-aded-e098c6fd2646
+objects:
+- apiVersion: v1
+  kind: Route
+  metadata:
+    annotations:
+      haproxy.router.openshift.io/timeout: 4m
+      template.openshift.io/expose-uri: http://{.spec.host}{.spec.path}
+    name: ${JENKINS_SERVICE_NAME}
+  spec:
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+    to:
+      kind: Service
+      name: ${JENKINS_SERVICE_NAME}
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      volume.beta.kubernetes.io/storage-class: ocs-storagecluster-ceph-rbd
+    name: ${JENKINS_SERVICE_NAME}
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: ${VOLUME_CAPACITY}
```

Comment:
The same way is used in this file for many other parms. Where are we sending the input here?
Please clarify